### ISSUE TITLE
Error 0 Failed to Extract Bootstrap

### ISSUE THREAD

**Issue by malhaar** [pwn20wndstuff/Undecimus#146]
Hello there,
I am constantly getting Error 0, I am switching from electra, iPhoneX 11.3.1. I tried jb version of Rollectra, its crashing a lot of times. then tried iPa version. that's rebooting a lot of times. 

so directly signed uncover v.2.0.2 and tried using MP Exploit, it restored rootFS successfully, then rebooted, then created snapshot successfully then rebooted. 

after that always Errorno: 0, Filename: viewcontroller.m Line: 2091 Function: installDeb

hope it helps.

Regards,

**Comment by Samgisaninja** [comment]
Glad you got it working then

**Comment by malhaar** [comment]
yeah .. also tried it on my iPad 10.5 11.3.1 same electra .. worked similar way .. this time I dropped Rollectra steps .. still it worked ..

**Comment by Supreme-Dong** [comment]
I have the exact same problem i'm on 11.4 B3, iPhone 8+
Here is my log: https://ghostbin.com/paste/2yh7s

**Comment by killerlooks10** [comment]
I would suggest you to watch this video. hope this will solve your problem
https://www.youtube.com/watch?v=0mdX-PcsZpk&t=195s
